src/SourceGenerators/System.Xaml/System.Xaml/XamlXmlReader.cs

```diff
@@ -30,99 +30,104 @@
 
 using Pair = System.Collections.Generic.KeyValuePair<Uno.Xaml.XamlMember,string>;
 
+using IsIncludedType = System.Func<string, string, System.Collections.Generic.KeyValuePair<bool?, bool>>;
+
 namespace Uno.Xaml
 {
 	public class XamlXmlReader : XamlReader, IXamlLineInfo
 	{
 		#region constructors
 
-		public XamlXmlReader (Stream stream)
-			: this (stream, (XamlXmlReaderSettings) null)
+		public XamlXmlReader (Stream stream, IsIncludedType isIncluded)
```

Comment:
Could we also use a default value instead of passing a complex lambda at all call sites?